### PR TITLE
Bug 1179227-[gui] allow to choose a repository using the wizard

### PR DESCRIPTION
**Pull request by MikeLing**

### CODE REVIEW

File: gui/mozregui/ui/nightlies.ui [hunk 1]
```diff
@@ -44,6 +47,26 @@
        </property>
       </widget>
      </item>
+     <item row="2" column="0">
+      <widget class="QLabel" name="label_3">
+       <property name="toolTip">
+        <string>the repository you want to select</string>
```
Comment (parkouss) [inline gui/mozregui/ui/nightlies.ui]: Can you change that with:

you can choose a specific repository name here

File: gui/mozregui/ui/nightlies.ui [hunk 2]
```diff
@@ -44,6 +47,26 @@
...
+       </property>
+      </widget>
+     </item>
+     <item row="2" column="1">
+      <widget class="QLineEdit" name="repository">
+       <property name="toolTip">
+        <string/>
+       </property>
+       <property name="placeholderText">
+        <string> choose a repository you want</string>
```
Comment (parkouss) [inline gui/mozregui/ui/nightlies.ui]: Same thing here:

you can choose a specific repository name here

**Comment by parkouss** [comment]
Yeah, excellent! This works really well, thanks. :) I tried that with the mozilla-aurora repo, this is good!
Please just change the text so the user will understand that this is not required to put something in the text field.

**Comment by parkouss** [comment]
Oh while we are here, could you add in the tooltip the examples we gave in the command line ?
[mozilla-aurora|mozilla-beta|...]
This will help the user to understand what can be put in the field.

Oh I love the text that we can see when there is no value in there!

**Comment by MikeLing** [comment]
Done :)

**Pull request merged**